kpi/deployment_backends/kobocat_backend.py

```diff
@@ -286,7 +286,7 @@ def redeploy(self, active=None):
                 'version': self.asset.version_id,
             })
         except KobocatDeploymentException as e:
-            if e.response.status_code == 404:
+            if hasattr(e, 'response') and e.response.status_code == 404:
```

Comment:
:facepalm: My bad!

And `if getattr(e, 'response', object()).__dict__.get('status_code') == 404` somehow doesn't have the same ring (or resolution of `@property`/`@getter`) to it :)